### DISCUSSION TITLE
Disable selection.

### DISCUSSION THREAD

**Discussion by isaacromerop** [sgratzl/lineup-lite#20] [Q&A]
I need to stop selecting the row based on any logic.

In my case, I have "status" which can be "Inactive", so when I click the "selectAll" button it won't select the row that don't meet requirement (status === "Inactive").

I manage to stop render the checkbox, but pressing the button in the header, will select all the rows.

Any idea?

**Comment by sgratzl** [comment]
it depends on whether you have the "featureRowSelect" activated or not (by default it is)

https://github.com/sgratzl/lineup-lite/blob/10d4760edaebfce04b1aac8cf362f1eadff3579d/packages/table/src/components/useLineUpLite.ts#L101C17-L118
